tests/test_batch/test_batch_jobs.py

```diff
@@ -399,7 +402,7 @@ def test_failed_job():
     _, _, _, iam_arn = _setup(ec2_client, iam_client)
 
     job_def_name = "exit-1"
-    commands = ["exit", "1"]
+    commands = ["kill"]
```

Comment:
Had to change the command because "exit 1" was raising a Docker Client exception. Instead, I think it would be best to run a command that returns an exit code > 1.